extensions/commands/cci/cmd_list_v2_ready.py

```diff
@@ -22,6 +27,7 @@ def list_v2_ready(conan_api: conan.api.conan_api.ConanAPI, parser, *args):
     parser.add_argument('path', help="Path to global recipes folder")
     parser.add_argument('-r', '--remote', action=OnceArgument, help="Remote to query against")
     parser.add_argument('-p', '--profiles', action='append', default=list(), help="List of profiles to run graph info with if --skip-binaries is false")
+    parser.add_argument('-f', '--output-file', action=OnceArgument, help="A JSON file path to store the output")
```

Comment:
We decided to use file redirection for this, not an extra argument, lets follow the Conan conventions.